spec/models/work_package/work_package_custom_fields_spec.rb

```diff
@@ -75,45 +75,49 @@ def self.change_custom_field_value(work_package, value)
       describe 'invalid custom field values' do
         context 'short error message' do
           shared_examples_for 'custom field with invalid value' do
-            let(:modified_work_package_subject) { 'Should not be saved' }
-
             before do
-              work_package.subject = modified_work_package_subject
-
               change_custom_field_value(work_package, custom_field_value)
             end
 
             describe 'error message' do
               before { work_package.save }
 
-              subject { work_package.errors[:custom_values] }
+              subject { work_package.errors["custom_field_#{custom_field.id}"] }
 
-              it { is_expected.to include(I18n.translate('activerecord.errors.messages.invalid')) }
+              it {
+                is_expected.to include(I18n.translate("activerecord.errors.messages.#{error_key}"))
+              }
             end
 
             describe 'work package attribute update' do
               subject { work_package.save }
 
-              it { is_expected.to be_falsey }
+              it { is_expected.to be_false }
```

Comment:
Indeed it is... What I **wanted** to do was `be false` (mind the space), however... After asking the internet I am all more ruby when I just ask for falsiness...